artman/pipelines/code_generation.py

```diff
@@ -23,6 +23,7 @@
 from artman.pipelines import pipeline_base
 from artman.tasks import io_tasks
 from taskflow.patterns import linear_flow
+from artman.tasks import emit_success
```

Comment:
This doesn't appear to be referenced in this file

Reply:
Done.